pepper-apis/config/DataDonationPlatform.yaml.ctmpl

```diff
@@ -29,9 +29,9 @@ network:
   name: managed
   instance_tag: study-server # necessary for firewall rules so that DSM will accept https requests from pepper
 {{if and (or (eq $environment "prod") (eq $environment "staging") (eq $environment "dev") (eq $environment "test")) (eq $gae "true") }}
-entrypoint: java -javaagent:tcell/tcellagent.jar -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_enable_heap_sampling=true,-cprof_service=pepper-backend,-cprof_cpu_use_per_thread_timers=true,-logtostderr -Xmx1640m -jar DataDonationPlatform.jar
+entrypoint: java -javaagent:tcell/tcellagent.jar -agentpath:/opt/cprof/profiler_java_agent.so=-cprof_enable_heap_sampling=true,-cprof_service=pepper-backend,-cprof_cpu_use_per_thread_timers=true,-logtostderr -Xmx1640m -Dio.grpc.netty.shaded.io.netty.transport.noNative=false -jar DataDonationPlatform.jar
```

Comment:
I discovered in last year's isolated CMI instance that a netty upgrade (which may or may not be required along with java17) broke things in really cryptic ways that took a few days to troubleshoot.  This may also be needed for cloud functions.